x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/spatial/20_geo_centroid.yml

```diff
@@ -52,3 +52,58 @@
   - match: { aggregations.centroid.location.lat: 51.00982965203002 }
   - match: { aggregations.centroid.location.lon: 3.9662131341174245 }
   - match: { aggregations.centroid.count: 6 }
+
+---
+"Test cartesian_centroid aggregation on shape field":
```

Comment:
Maybe we can rename this file to 10_centroid so it can contain geo and cartesian implementations?